src/pages/cart/CartCard.jsx

```diff
@@ -25,7 +25,11 @@ export function CartCard({value}){
                     <button onClick={()=>updateItemQuantity(_id, dispatch, "increment")}><i className="fas fa-plus"></i></button>
                 </div>
                 <div className="card-btn-wrapper">
-                    <button className="btn btn-primary-outline">SAVE FOR LATER</button>
+                    <button className="btn btn-primary-outline" 
+                    onClick={()=>{
+                        state.wishlist.filter(item=> item._id===_id).length!==1 && addToWishlist(value, dispatch)
```

Comment:
you can use find method here also

Reply:
Thanks for the suggestion, I have updated  it.